packages/cli/src/workflows/workflow-execution.service.ts

```diff
@@ -147,6 +148,35 @@ export class WorkflowExecutionService {
 			triggerToStartFrom,
 		};
 
+		/**
+		 * Historically, manual executions in scaling mode ran in the main process,
+		 * so some execution details were never persisted in the database.
+		 *
+		 * Currently, manual executions in scaling mode are offloaded to workers,
+		 * so we persist all details to give workers full access to them.
+		 */
+		if (
+			config.getEnv('executions.mode') === 'queue' &&
+			process.env.OFFLOAD_MANUAL_EXECUTIONS_TO_WORKERS === 'true'
+		) {
```

Comment:
Ideally, this additiona should not be needed and all executions should always persist all their data, but doing so at this time might have unexpected side effects, so I found it safer to persist this data only when I know for certain it will be needed.

Reply:
Could we simplify and always store these in the DB no matter what the mode is?